src/ui/components/results/paginationcomponent.js

```diff
@@ -239,27 +253,32 @@ export default class PaginationComponent extends Component {
     }
 
     return {
-      ellipses: {
+      pinnedNumbers: {
```

Comment:
This is kind of confusing. What's the difference between pinnedNumbers and ellipses?

Reply:
ellipses will only show if there's a gap between the last page number and the pinned page number